### PR TITLE
feat(Author): email author variant

### PR DESCRIPTION
**Pull request by ovbm**
Adds new Large Variant to Email Author: 

Screenshot 2023-10-17 at 17 32 01

ToDo:
-[] Test Mailchimp

### CODE REVIEW

File: apps/publikator/components/editor/modules/author/ui.js [hunk 1]
```diff
@@ -33,6 +34,7 @@ export default ({ TYPE, newBlock, editorOptions }) => {
     if (!value.blocks.some(matchBlock(TYPE))) {
       return null
     }
+    const handlerFactory = createOnFieldChange(onChange, value)
```
Comment (lucianak) [inline apps/publikator/components/editor/modules/author/ui.js:37]: pedantic but for me `handlerFactory` is not a very clear name, except if it's some kind of standard?

File: apps/publikator/components/editor/modules/author/ui.js [hunk 2]
```diff
@@ -50,6 +52,8 @@ export default ({ TYPE, newBlock, editorOptions }) => {
               portrait: author.value.portrait,
               credentials: author.value.credentials,
             },
+            isLarge: author.value.isLarge,
```
Comment (lucianak) [inline apps/publikator/components/editor/modules/author/ui.js:55]: just to be sure I understand correctly: `isLarge` and `greeting` are independent of each other, right? `isLarge` can be false and the component still displays the greeting

**Review by lucianak — COMMENTED** [review]
Looks good to me, did you check how it looks on mailchimp?

File: apps/publikator/components/editor/modules/author/ui.js (hunk 1 above)
Comment (ovbm) [reply apps/publikator/components/editor/modules/author/ui.js:37]: Yes, I found it as a standard way to create Field onChange Events (see links)

File: apps/publikator/components/editor/modules/author/ui.js (hunk 2 above)
Comment (ovbm) [reply apps/publikator/components/editor/modules/author/ui.js:55]: they are independent of each other, however, the greeting is only displayed in the "isLarge" case

**Pull request merged**